docs/FAQ/General-FAQ.md

```diff
@@ -1,5 +1,20 @@
 # General FAQ
 
+## How and what commit is chosen when running `evergreen patch`?
+
+The Evergreen CLI grabs the user's config file and determines what project should be used. The CLI then establishes a connection with the Evergreen server and retrieves that project's information, specifically the GitHub org + repo + branch (which we'll call, org, repo, and main respectively).
```

Comment:
Can we specify that this is referring to manual patches? 

Also should we add some words about pr patches?